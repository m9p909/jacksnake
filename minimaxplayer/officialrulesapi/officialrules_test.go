package officialrulesapi_test

import (
	"github.com/gkampitakis/go-snaps/snaps"
	. "jacksnake/minimaxplayer/officialrulesapi"
	"testing"
)

func Test_GetValidMoves(t *testing.T) {
	state := getRulesState()

	rules := OfficialRulesImpl{}
	res := rules.GetValidMoves(state, "1")
	snaps.MatchSnapshot(t, res)
}
