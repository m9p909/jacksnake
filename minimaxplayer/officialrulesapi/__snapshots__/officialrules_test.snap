
[Test_GetValidMoves - 1]
[]string{"down", "left"}
---
